### ISSUE TITLE
Unmet dependencies (bs4 and lxml)

### ISSUE THREAD

**Issue by Blogem** [fondberg/spotcast#38]
I installed spotcast trough hacs. After installing and configuring, I got the following error when calling spotcast.start: No module named 'bs4'

After installing BeautifulSoup4 in the virtualenv, I received the following error: bs4.FeatureNotFound: Couldn't find a tree builder with the features you requested: lxml. Do you need to install a parser library?

After installing this one in the virtualenv and restarting homeassistant, everything is working.

I kinda expected that these dependencies would be solved by hacs or otherwise get mentioned in the readme.

Everything is working now, so I'm happy :).

**Comment by fondberg** [comment]
Weird that a non imported file should make this happen. I have now removed it and bs4 should no longer be necessary

**Comment by fondberg** [comment]
Please try again with version v2.5

**Comment by soldag** [comment]
I had the same issue. Seems to be caused by missing dependencies in https://github.com/soldag/spotify-webplayer-token/blob/master/setup.py.

I fixed this in https://github.com/enriquegh/spotify-webplayer-token/pull/3. After the PR has been merged and released, version of `spotify_token` dependency needs to be bumped here.

**Comment by fondberg** [comment]
So the token library needed the dependency which it hadn't in the version spotcast listed?

**Comment by soldag** [comment]
Yes, right. They are actually listed in [requirements.txt](https://github.com/enriquegh/spotify-webplayer-token/blob/master/requirements.txt), but not in [setup.py](https://github.com/enriquegh/spotify-webplayer-token/blob/master/setup.py), which is taken into account when installing it via pip.

**Comment by rhannink** [comment]
Hallo,

I did:

```
source /srv/homeassistant/bin/activate
pip3 install lxml
pip3 install BeautifulSoup4
```

But Home Assistant is still complaining about the passer: `bs4.FeatureNotFound: Couldn't find a tree builder with the features you requested: lxml. Do you need to install a parser library?`

Any idea why it is still complaining? Python 3.7 on hassbian and HA 0.102 installeer.

**Comment by fondberg** [comment]
@soldag until the PR is merged in spotify_token, do you have a suggestion on how to deal with this?

**Comment by soldag** [comment]
You could add the missing requirements to the manifest of spotcast again as a workaround until the issue is fixed in `spotify_token`.

Don't know why installing them manually didn't fix the error for @rhannink, though :man_shrugging:

**Comment by enriquegh** [comment]
@fondberg @soldag pinged you in the other thread but I've released 0.1.2 with soldag's fix.
Thanks again!

**Comment by fondberg** [comment]
Thanks @enriquegh !
